recipe/meta.yaml

```diff
@@ -53,13 +53,15 @@ requirements:
     - charset-normalizer
     - conda-libmamba-solver >=23.11.0
     - conda-package-handling >=2.2.0
+    - distro >=1.5.0
     - jsonpatch >=1.32
-    - menuinst >=1.4.11,<2         # [win]
+    - menuinst
```

Comment:
Thanks for the feedback :) 

I don't know how to answer to the first question, since we haven't conducted surveys. What we did here is to extend the standard behaviour for Windows, so it is now applicable to all the platforms. In that vein, a `shortcuts` config option is now available too (defaults to `yes`). Folks can add a `shortcuts: false` item in their config, or `--no-shortcuts` their commands. Workflow tools can export `CONDA_SHORTCUTS=false` if they get in the way.

I don't know how "unexpected" this behavior is, but it has been designed to be reversible (uninstall the package, and the shortcut goes away).

If you are suggesting we default to `shortcuts: false` in Unix, that might be a barrier of adoption for application users that would expect a shortcut. If the community wants that (and I would have expected such a comment in the long review process the CEP went through, or maybe in the two-year-old PR 😂), we can adjust it but it'll need a new release. I don't think it's too problematic right now given how there are virtually no menuinst v2 shortcuts in the wild (only napari and spyder, and they implement it via separate packages).

Hope that clarifies the concerns.

Reply:
Thanks, I wanted to make sure my hunch about the default was right.
Yes, my timing on this is not good.
I didn't review the PR to `conda` and hence didn't realize this beforehand.
I don't think(/remember) we explicitly discussed `conda`'s default in the CEP but just the requirements for correctly/uniformly handling this for implementations that support it.
It's just that it *effectively* is like flipping the default from `false` to `true` (which I'm personally impartial to) -- for people that never use `conda` on Windows it can be unexpected (good or bad) to suddenly see this.

But my concern is rather regarding that I'm not aware when/if we announced this change; with such an announcement covering of course both points:
1. Letting people know of `CONDA_SHORTCUTS=false` etc. for automated workflows,
2. advertise the new capabilities ;).

----
Good to know that `napari`/`spyder` use the separate-package strategy.
If I find the time (but not sure I will..), I might look into `conda`'s messaging around this to see how things are behaving now.

Thanks for confirming/explaining how things are implemented.

Reply:
Do we have documentation on these options somewhere?

Reply:
Looking at the packages in conda-forge more, it appears many of them only create shortcuts for Windows currently (probably influenced in no small part to the fact `menuinst` was for a long time Windows only). Admittedly packages may start adding macOS and Linux shortcuts once the option is available. Though that likely will happen over some time. So don't expect that just changing the default in Conda will have an immediate effect in the sense that not much will change for users right away (as packages will still need to adopt it)

Reply:
The release notes have been updated to include additional info on menuinst support (see https://github.com/conda/conda/releases/tag/23.11.0). This will also be mentioned in the upcoming blog post.